src/build_translations.py

```diff
@@ -140,14 +146,14 @@ def nonempty(seq):
         when_distance, _ = lang_blocks["when distance < %n"].split(" < %n")
 
     end_block = None
-    for translated_text, english_spec in extra_aliases.items():
+    for translated_text, english_spec in copy(extra_aliases).items():
         if english_spec == "end":
             end_block = translated_text
             del extra_aliases[translated_text]
```

Comment:
Because we are potentially deleting keys from the object while looping through it, which will crash python.